primitives/src/runtime_api.rs

```diff
@@ -123,7 +123,7 @@ use sp_std::{collections::btree_map::BTreeMap, prelude::*};
 
 sp_api::decl_runtime_apis! {
 	/// The API for querying the state of parachains on-chain.
-	#[api_version(2)]
+	#[api_version(4)]
```

Comment:
We still shouldn't touch the base version because there are runtimes (Polkadot) at version 2. Let's keep the api declaration unchanged and only update the version Kusama implements.

Reply:
I'm writing this with the impression we are releasing just for Kusama. Ignore if this is not the case.